### PR TITLE
Automate docs generation

### PR DESCRIPTION
**Pull request by AlexDenisov**

### CODE REVIEW

File: docs/generated/CLIOptions.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
+--workers number		How many threads to use
+
+--dry-run		Skips real mutants execution. Disabled by default
```
Comment (stanislaw) [inline docs/generated/CLIOptions.rst]: indentation?

File: docs/conf.py [hunk 2]
```diff
@@ -14,17 +14,25 @@
...
 
+import re
 import guzzle_sphinx_theme
 
 # -- Project information -----------------------------------------------------
 
-project = 'Mull'
-copyright = '2019, Alex Denisov'
-author = 'Alex Denisov'
+def project_version():
```
Comment (stanislaw) [inline docs/conf.py]: Great stuff

File: docs/generated/CLIOptions.rst (hunk 1 above)
Comment (stanislaw) [reply docs/generated/CLIOptions.rst]: though you probably don't have control over this

Comment (AlexDenisov) [reply docs/generated/CLIOptions.rst]: It's automatically generated. We shouldn't care about the prettiness of the contents as long as sphinx can handle it correctly.

**Pull request merged**